core/src/server/rpc_module.rs

```diff
@@ -552,6 +557,11 @@ impl<Context: Send + Sync + 'static> RpcModule<Context> {
 		)
 	}
 
+	/// Removes method if it exists.
+	pub fn remove_method(&mut self, method_name: &'static str) -> Option<MethodCallback> {
```

Comment:
I would want add something to actually know whether it's a method call or subscription that was removed.

If call was a subscription the `unsubscribe method` should be removed well because it will be "dangling"... but that is not easy to implement currently.

Thus, I think the way forward is to clearly document that it's the caller's responsible to remove both for methods subscriptions i.e, both the "subscribe method" and "unsubscribe method" which is acceptable to me.